src/TodoList.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+
+function TodoList (props) {
+  
```

Comment:
missing array destructuring like const {todoList} = props then in the jsx you can just reference the todoList.